gm4_soul_fragments/data/gm4_soul_fragments/functions/soul_fragment/seek_player.mcfunction

```diff
@@ -0,0 +1,16 @@
+#@s = soul fragment with a linked player nearby (@e[type=minecraft:area_effect_cloud,tag=gm4_soul_fragment,distance=..16])
+#from player/search_fragment
+
+#emit particles near linked player
+particle minecraft:soul ~ ~ ~ .05 0 .05 .015 1 force
+
+#hum when near linked player
+playsound minecraft:block.beacon.ambient master @a[distance=..22] ~ ~ ~ .6 .4
```

Comment:
Do not play sounds on master. I'd personally put this under `player` but that is up for debate.